index.html

```diff
@@ -0,0 +1,54 @@
+<html>
+  <head>
+    <title>Plan My Trek</title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="foundation.css">
+    <link href="styles.css" media="screen" rel="stylesheet" type="text/css">
+  </head>
+  <body>
+    <header>
+      <div class="row">
+        <div class="column">
+          <h1>Embark on the adventure of a lifetime.</h3>
+        </div>
+      </div>
+    </header>
+
+    <!--Centering button from: http://foundation.zurb.com/forum/posts/1115-centering-a-button-group-->
+    <div class="row">
+      <div class="small-10 medium-4 large-3 large-centered medium-centered small-centered columns">
+        <div class="large button landing-button" id="load"> See trips </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column">
+        <article class="callout" id="displayed-trip"></article>
+      </div>
+    </div>
+    <div class="row small-up-1 medium-up-3" id="trips">
+      <div class="columns tile asia" >
+        <h3>Asia</h3>
+      </div>
+      <div class="columns tile africa">
+        <h3>Africa</h3>
+      </div>
+      <div class="columns tile europe">
+        <h3>Europe</h3>
+      </div>
+      <div class="columns tile north-america">
+        <h3>North America</h3>
+      </div>
+      <div class="columns tile south-america">
+        <h3>South America</h3>
+      </div>
+      <div class="columns tile antarctica">
+        <h3>Antarctica</h3>
+      </div>
+      <div class="columns tile australia">
+        <h3>Australia</h3>
+      </div>
+    </div>
+    <script type="text/javascript" src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
+    <script type="text/javascript" src="index.js"></script>
+  </body>
+</html>
```

Comment:
Gah!  no Semantic HTML  Div Crazyness!  ARG!

Seriously it's ok, but would be better with more meaningful HTML tags.